index.html

```diff
@@ -10,6 +10,11 @@
     <title>HTML & CSS Training</title>
   </head>
   <body>
-      <h1 class="title">Hello World</h1>
+      <div>
+        <h1 class="book-title">Título del libro <p class="book-genre">(género)</p></h1>
```

Comment:
use span instead of p.  span is to define classes to text inside another text element